bazel/repository_locations.bzl

```diff
@@ -76,8 +76,8 @@ REPOSITORY_LOCATIONS = dict(
         urls = ["https://github.com/google/protobuf/archive/v3.5.0.tar.gz"],
     ),
     envoy_api = dict(
-        commit = "2dcc435e8ae1d35f8c3a4fa9f132778482fb1a78",
-        remote = "https://github.com/envoyproxy/data-plane-api",
+        commit = "1ac9588432fc095c7a6e8342304f3a40523e9927",
```

Comment:
please update now that your change has been merged

Reply:
Updated to latest sha.